code/obj/storage/secure_closets.dm

```diff
@@ -586,11 +586,14 @@
 	/obj/item/storage/backpack/satchel/medic,
 	/obj/item/storage/backpack/robotics,
 	/obj/item/storage/backpack/genetics,
+	/obj/item/storage/backpack/pathology,
 	/obj/item/storage/backpack/satchel/robotics,
 	/obj/item/storage/backpack/satchel/genetics,
+	/obj/item/storage/backpack/satchel/pathology,
 	/obj/item/storage/box/clothing/medical,
 	/obj/item/storage/box/clothing/geneticist,
 	/obj/item/storage/box/clothing/roboticist,
+	/obj/item/storage/box/clothing/pathologist,
```

Comment:
The job isn't even around. Hold off on adding these if and when pathology gets brought back.

Reply:
Should be good now